src/components/Feedbacks/Item/styles.ts

```diff
@@ -0,0 +1,51 @@
+import styled from "styled-components";
+
+import { IItem } from "../../../models/item.model";
+
+export const ItemContainer = styled.div<IItem>`
+  width: 600px;
+  height: 360px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)),
+    url(${(props) => props.backgroundImg});
+  margin: 5px;
+  border-radius: 8px;
+  color: #fff;
+`;
+
+export const HandleContent = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 20px;
+`;
+
+export const UserText = styled.p`
```

Comment:
Aqui o ideal era usar blockquote/quote, pois é a opinião de alguém. Inclusive, note que no seu layout, faltaram as aspas.

https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/

![image](https://user-images.githubusercontent.com/3991845/126393189-28c2b584-be4b-459a-a1b7-9d73bee979e9.png)